app/models/manageiq/providers/hawkular/middleware_manager/event_catcher/runner.rb

```diff
@@ -1,6 +1,11 @@
 class ManageIQ::Providers::Hawkular::MiddlewareManager::EventCatcher::Runner <
   ManageIQ::Providers::BaseManager::EventCatcher::Runner
-  INCLUDE_EVENTS = {'Hawkular Deployment' => '.*'}.freeze
+
+  # INCLUDE only supported events. Filter out any others.
+  # - event category must eq a hash key
+  # - event message must match hash value :pattern, if supplied
+  # - hash value :type specifies the entity type for which the event's canonical path must have an entry
+  INCLUDE_EVENTS = {'Hawkular Deployment' => {:pattern => '.*', :type => 'MiddlewareServer'}}.freeze
```

Comment:
The advantage of this structure is that knowledge is encoded for each event type in one place.  The proposed change would require updates in two places for one event type.  Still, I can see that it is a dense structure. In the next revision I will break it into two maps, but both declared in Runner.  I will eliminate EventParser, it unnecessarily complicates/distributes processing logic.